src/core/libmaven/Scan.h

```diff
@@ -1,6 +1,10 @@
 #ifndef SCAN_H
 #define SCAN_H
 #include "mzSample.h"
+#include <QRegExp>
+#include <QString>
+#include <QStringList>
+
```

Comment:
unnecessary includes I guess. did you forget to remove them ?

Reply:
@rish9511 No, it is being used by Scan class.

Reply:
I just compiled the branch without these includes and it compiled fine

Reply:
@rish9511 It is failing on Travis. Not on my system also.